app/store/index.js

```diff
@@ -9,9 +9,13 @@ const chromep = new ChromePromise()
 
 export const LANGUAGES = [
   { key: 'GERMAN', value: 'GER' },
-  { key: 'SPANISH', value: 'SPA' }
+  { key: 'SPANISH', value: 'SPA' },
```

Comment:
We could also try updating these values to the ISO code (which I think the Chrome TTS is using?), and then using that as the `lang` value in the `speak` function.